scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl

```diff
@@ -286,8 +286,8 @@ task AssertCostIsTrackedAndExpected {
 
         echo "project_id = ~{project_id}" > ~/.bigqueryrc
         bq --apilog=false query --project_id=~{project_id} --format=csv --use_legacy_sql=false \
-            'SELECT call, step, event_key, sum(event_bytes)
-              FROM `~{dataset_name}.cost_observability`
+            'SELECT call, step, event_key, sum(event_bytes) FROM
+                (SELECT distinct step, call, event_key, event_bytes, shard_identifier FROM `~{dataset_name}.cost_observability`)
               GROUP BY call, step, event_key
               ORDER BY call, step, event_key' > output/cost_observability.csv
```

Comment:
tl;dr I don't understand what this query is trying to do, happy to discuss faces in standup. It seems that the `GROUP BY` is squashing out the notion of shards?

Reply:
Query is rewritten. Closing this thread.